src/AElf.CSharp.CodeOps/Validators/Whitelist/IWhitelistProvider.cs

```diff
@@ -48,6 +48,8 @@ private void WhitelistAssemblies(Whitelist whitelist)
             .Assembly(typeof(IMethod).Assembly, Trust.Full) // AElf.CSharp.Core
             .Assembly(typeof(SecretSharingHelper).Assembly, Trust.Partial) // AElf.Cryptography
             .Assembly(typeof(ISmartContractBridgeContext).Assembly, Trust.Full) // AElf.Kernel.SmartContract.Shared
+            .Assembly(typeof(Groth16.Net.Verifier).Assembly, Trust.Full) // AElf.Cryptography.ECDSA
```

Comment:
This is for supporting zklogin feature, it seems wired to put this upload on this PR, but not a big problem.

Reply:
Yes, this is for supporting zklogin feature, I will roll back these codes if they cannot be verified in time.